docs/tutorials/analysis/1D/spectral_analysis_rad_max.ipynb

```diff
@@ -0,0 +1,527 @@
+    "The gamma-astro-data-format specifications offer two different ways to store this information:\n",
+    "* if the same $\\theta$ cut is applied at all energies and offsets, [a `RAD_MAX` keyword](https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/#rad-max) is added to the header of the data units containing IRF components. This should be used to define the size of the ON and OFF regions (i.e. the radius of the `~regions.CircleSkyRegion` that we will use to define the Dataset geometry);\n",
+    "* in case an energy- (and offset-) dependent $\\theta$ cut is applied, its values are stored in additional `FITS` data unit, named [`RAD_MAX_2D`](https://gamma-astro-data-formats.readthedocs.io/en/latest/irfs/point_like/#rad-max-2d). `Gammapy` contains a class to automatically read this table, `~gammapy.irf.RadMax2D`. In this notebook we will focus on how to modify the spectral reduction in order to use it. We remark that in this case a `~regions.PointSkyRegion` (and not a `~regions.CircleSkyRegion`) should be used to define the ON region. If a geometry based on a `~regions.PointSkyRegion` is feeded to the spectra and the background `Maker`s, the latter will automatically use the values stored in the `RAD_MAX_2D` table.\n",
+    "\n",
+    "Beside the definition of the ON region during the data reduction, the remaining steps are identical to the other [1D spectral analysis example](spectral_analysis.ipynb), so we will not detail the data reduction steps already presented in the other tutorial.\n",
+    "\n",
+    "**Objective: perform the data reduction and analysis of 2 Crab Nebula observations of MAGIC and fit the resulting datasets.**\n",
+    "\n",
+    "## Introduction\n",
+    "\n",
+    "We load two MAGIC observations in the [gammapy-data](https://github.com/gammapy/gammapy-data) containing IRF component with a $\\theta$ cut. In this example, we directly use the `~gammapy.data.Observation` class to read them, instead of a `~gammapy.data.DatStore`.\n",
+    "\n",
+    "We define the ON region, this time as a `PointSkyRegion` instead of a `CircleSkyRegion`, i.e. we specify only the center of our ON region. We create a `RegionGeom` adding to the region the estimated energy axis of the `~gammapy.datasets.SpectrumDataset` object we want to produce. The corresponding dataset maker will automatically use the $\\theta$ values in `~gammapy.irf.RadMax2D` to set the appropriate ON region sizes (based on the offset on the observation and on the estimated energy binning).\n",
+    "\n",
+    "We define the Maker object that will extract the OFF counts from the reflected regions in the field of view. As for the `SpectrumDataSetMaker`, the `~gammapy.irf.RadMax2D` will be used to define the sizes of the OFF regions.\n",
+    "\n",
+    "Once the datasets with the ON and OFF counts are created, we can perform a 1D likelihood fit, exactly as illustrated in the previous example."
+   ]
```

Comment:
In the context you might insist on the general difference:
full containment IRFs having no theta cut, they require applying some correction if one wants to correct for flux leaking out of the ON region.
pointlike IRFs, because they are designed applying a theta cut, directly provide an effective area taking into account the effect of leakage.

Reply:
Stressed the difference.